torchvision/models/detection/_utils.py

```diff
@@ -507,3 +508,26 @@ def _topk_min(input: Tensor, orig_kval: int, axis: int) -> int:
     axis_dim_val = torch._shape_as_tensor(input)[axis].unsqueeze(0)
     min_kval = torch.min(torch.cat((torch.tensor([orig_kval], dtype=axis_dim_val.dtype), axis_dim_val), 0))
     return _fake_cast_onnx(min_kval)
+
+
+def _box_loss(
+    type: str,
+    box_coder: BoxCoder,
+    anchors_per_image: Tensor,
+    matched_gt_boxes_per_image: Tensor,
+    bbox_regression_per_image: Tensor,
+    cnf: Optional[Dict[str, float]] = None,
```

Comment:
nit: I would use a `**kwargs` here instead, but it's personal taste

Reply:
Yeap you are absolutely right. That's what I would love to use but JIT doesn't like it. :(

Reply:
Understood. Maybe sometime in the near future we won't need to bother about torchscript anymore. But we are not yet there